### DISCUSSION TITLE
Podman support

### DISCUSSION THREAD

**Discussion by cfouche3005** [pelican-dev/panel#273] [Ideas]
Podman is an alternative to Docker with some features that Docker lacks and is more secure by default.
It would be cool for Pelican to add support for podman

**Comment by lancepioch** [comment]
Thanks, this is planned, but will take a while!

**Reply by linux-universe** [reply]
any updates on this?

**Reply by ALameLlama** [reply]
for what it's worth, I ended up getting this working on Alma8 using podman podman-docker and then updating my wing to have the k8s-file logging driver since local doesn't exist in podman
```
  log_config:
    type: k8s-file
```
